sections/deploying.md

```diff
@@ -117,6 +151,20 @@
   - Multi-Container Docker
   - Preconfigured Docker
 
+- If not supported, you can write your custom platform.
+
+### Elastic Beanstalk vs CloudFormation
```

Comment:
can you please make it short and convert it to a table?

example: `sections/other_compute.md(  Batch vs Lambda )`

Reply:
Done